<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SelectItemsPopup</class>
 <widget class="QDialog" name="SelectItemsPopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>537</width>
    <height>290</height>
   </rect>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_5">
   <item>
    <spacer name="spacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>0</width>
       <height>15</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>A&amp;vailable</string>
         </property>
         <property name="buddy">
          <cstring>listAvailable</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listAvailable">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonSelect">
         <property name="arrowType">
          <enum>Qt::RightArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonUnselect">
         <property name="arrowType">
          <enum>Qt::LeftArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Sele&amp;cted</string>
         </property>
         <property name="buddy">
          <cstring>listSelected</cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QListWidget" name="listSelected">
         <property name="editTriggers">
          <set>QAbstractItemView::NoEditTriggers</set>
         </property>
         <property name="showDropIndicator" stdset="0">
          <bool>false</bool>
         </property>
         <property name="dragDropMode">
          <enum>QAbstractItemView::InternalMove</enum>
         </property>
         <property name="alternatingRowColors">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <spacer name="verticalSpacer_3">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QToolButton" name="buttonUp">
         <property name="arrowType">
          <enum>Qt::UpArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonDown">
         <property name="arrowType">
          <enum>Qt::DownArrow</enum>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="verticalSpacer_4">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Apply|QDialogButtonBox::Cancel</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>listAvailable</tabstop>
  <tabstop>listSelected</tabstop>
  <tabstop>buttonSelect</tabstop>
  <tabstop>buttonUnselect</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonSelect</sender>
   <signal>clicked()</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>selectItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>263</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>203</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUnselect</sender>
   <signal>clicked()</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>unselectItem()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>159</y>
    </hint>
    <hint type="destinationlabel">
     <x>515</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listAvailable</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>selectItem(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>124</x>
     <y>45</y>
    </hint>
    <hint type="destinationlabel">
     <x>115</x>
     <y>0</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>listSelected</sender>
   <signal>doubleClicked(QModelIndex)</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>unselectItem(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>377</x>
     <y>96</y>
    </hint>
    <hint type="destinationlabel">
     <x>383</x>
     <y>4</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>clicked(QAbstractButton*)</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>buttonBoxClicked(QAbstractButton*)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>262</x>
     <y>258</y>
    </hint>
    <hint type="destinationlabel">
     <x>262</x>
     <y>140</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonDown</sender>
   <signal>clicked()</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>moveItemDown()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>153</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonUp</sender>
   <signal>clicked()</signal>
   <receiver>SelectItemsPopup</receiver>
   <slot>moveItemUp()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>513</x>
     <y>124</y>
    </hint>
    <hint type="destinationlabel">
     <x>268</x>
     <y>144</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>selectItem(QModelIndex)</slot>
  <slot>unselectItem(QModelIndex)</slot>
  <slot>selectItem()</slot>
  <slot>unselectItem()</slot>
  <slot>buttonBoxClicked(QAbstractButton*)</slot>
  <slot>moveItemUp()</slot>
  <slot>moveItemDown()</slot>
 </slots>
</ui>
